### PR TITLE
Added outside input.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: app/src/components/inputs/outsideInput/outsideInput.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    onBlur: () => {},
+    onKeyUp: () => {},
+    refFunction: () => {},
+    active: false,
+    touched: false,
+    error: '',
+  };
+
+  state = {
+    passwordShown: false,
```
Comment (DiscoElevator) [inline app/src/components/inputs/outsideInput/outsideInput.jsx]: passwordVisible

File: app/src/components/inputs/outsideInput/outsideInput.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  };
+  getInputType = () => {
+    if (this.props.type !== 'password') {
+      return this.props.type;
+    }
+    return this.state.passwordShown ? 'text' : 'password';
+  };
+  showPass = (e) => {
```
Comment (DiscoElevator) [inline app/src/components/inputs/outsideInput/outsideInput.jsx]: showPassword

File: app/src/components/inputs/outsideInput/outsideInput.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    e.preventDefault();
+    !this.state.passwordShown && this.setState({ passwordShown: true });
+  };
+  hidePass = (e) => {
```
Comment (DiscoElevator) [inline app/src/components/inputs/outsideInput/outsideInput.jsx]: hidePassword

File: app/src/components/inputs/outsideInput/outsideInput.scss [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    .input {
+      padding-right: 35px;
+    }
+    .eye-icon {
+      position: absolute;
+      top: 0;
+      right: 5px;
+      height: 38px;
+      width: 30px;
+      background-image: url(./img/pass-hided.svg);
```
Comment (DiscoElevator) [inline app/src/components/inputs/outsideInput/outsideInput.scss]: password-hidden

File: app/src/components/inputs/outsideInput/outsideInput.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      background-repeat: no-repeat;
+      background-size: 60%;
+      background-position: center;
+
+      &.opened {
+        background-image: url(./img/pass-shown.svg);
```
Comment (DiscoElevator) [inline app/src/components/inputs/outsideInput/outsideInput.scss]: password-visible

File: app/src/components/inputs/outsideInput/outsideInput.jsx (hunk 1 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/outsideInput/outsideInput.jsx]: fixed

File: app/src/components/inputs/outsideInput/outsideInput.jsx (hunk 2 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/outsideInput/outsideInput.jsx]: fixed

File: app/src/components/inputs/outsideInput/outsideInput.jsx (hunk 3 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/outsideInput/outsideInput.jsx]: fixed

File: app/src/components/inputs/outsideInput/outsideInput.scss (hunk 4 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/outsideInput/outsideInput.scss]: fixed

File: app/src/components/inputs/outsideInput/outsideInput.scss (hunk 5 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/outsideInput/outsideInput.scss]: fixed

**Pull request merged**